docs/versioned_docs/version-v1.9/guides/migrating-from-mitreid.mdx

```diff
@@ -0,0 +1,27 @@
+---
+id: migrating-from-MITREid
+title: Migrating from MITREid
+---
+
+**Please note this page doesn't pretend to be a step-by-step guide**.
+This piece of documentation will try to give you some hints to move your current MITREid instance to Hydra. 
+
+#### Scopes
+Scopes in hydra doesn't have a relational entity, every client defines its own array of scopes.
+
+#### Scope vs Scp
+Since hydra uses scp as the scope claim, you have to be sure your client libraries are capable to use both, we had to modify the behavior of our version of spring.security.oauth2 library following the example: [Custom claim](https://dev.to/toojannarong/spring-security-with-jwt-the-easiest-way-2i43)
```

Comment:
```suggestion

Since hydra uses scp as the scope claim, you have to be sure your client libraries are capable to use both, we had to modify the behavior of our version of spring.security.oauth2 library following the example: [Custom claim](https://dev.to/toojannarong/spring-security-with-jwt-the-easiest-way-2i43)
```